bin/proxy.js

```diff
@@ -42,12 +67,14 @@ function HostsInitialize(region) {
 }
 
 function HostsClean(region) {
-    if (region && region.platform !== 'console') {
+    // Only clean hosts file on Windows as other platforms (for now) have to do all hosts edits manually
```

Comment:
Not exacly true. If proxy was run as root and the hosts modified, you'd need to remove the added entries here, too. That's why I added the weird `cannotWrite` [function](https://github.com/caali-hackerman/tera-proxy/pull/1/files#diff-ad8977bbefd58040b0b0f6ad0ec97159R104) and stored it in a [module global](https://github.com/caali-hackerman/tera-proxy/pull/1/files#diff-07a345d6fd643ba2bc471a507d7cbd68R115) so I could later determine whether I need to remove them on exit.

That's of course a subpar solution, but I didn't want to influence too much of the code structure just for this.